frontend/public/components/utils/dropdown.jsx

```diff
@@ -417,12 +417,14 @@ export class Dropdown extends DropdownMixin {
       titlePrefix,
       describedBy,
       disabled,
+      isValid,
```

Comment:
you don't need this prop if you put your logic to detect duplicate entry out of DropdownField.

Reply:
Again, this is intended to add an inline visual indicator of unexpected selection, before being submitted. This can be used to make the dropdown show an error state inline, if any such validation is required.